src/implementations/vanilla/components/text-area/_text-area.scss

```diff
@@ -0,0 +1,110 @@
+/**
+ * Textarea component
+ * @define text-area
+ */
+
+@use "sass:map";
+@use '@ecl/theme-dev/theme';
+@use '@ecl/vanilla-layout-grid/mixins/breakpoints';
+
+$_border-radius: null !default;
+$_border-color: null !default;
+$_border-color-hover: null !default;
+$_border-color-focus: null !default;
+$_border-color-invalid: null !default;
+$_border-color-invalid-hover: null !default;
+$_border-width: 1px;
+$_border-width-focus: null !default;
+$_border-width-focus: 3px;
+$_color-input: null !default;
+$_color-placeholder: null !default;
+$_shadow: null !default;
+$_shadow-hover: null !default;
+$_padding-vertical: map.get(theme.$spacing, 's');
+$_padding-horizontal: null !default;
+
+.ecl-text-area {
+  appearance: none;
+  background-color: map.get(theme.$color, 'white-100');
+  border: $_border-width solid $_border-color;
+  border-radius: $_border-radius;
+  box-shadow: $_shadow;
+  box-sizing: border-box;
+  color: $_color-input;
+  display: block;
+  font: map.get(theme.$font, 'm');
+  margin: 0;
+  max-width: 100%;
+  padding: calc(#{$_padding-vertical} - #{$_border-width})
+    calc(#{$_padding-horizontal} - #{$_border-width});
+
+  &::placeholder {
+    color: $_color-placeholder;
+    font: map.get(theme.$font, 'm');
+    opacity: 1;
+  }
+
+  &:hover {
+    border-color: $_border-color-hover;
+    box-shadow: $_shadow-hover;
+  }
+
+  &:focus,
+  &:active {
+    border-color: $_border-color-focus;
+    border-width: $_border-width-focus;
+    outline: none;
+    padding: calc(#{$_padding-vertical} - #{$_border-width-focus})
+      calc(#{$_padding-horizontal} - #{$_border-width-focus});
+  }
+}
+
+.ecl-text-area--m {
+  width: 100%;
+
+  @include breakpoints.up('m') {
+    width: 50%;
+  }
+
+  @include breakpoints.up('l') {
+    width: map.get(theme.$form, 'm');
```

Comment:
didn't find anything in the specs in abstract about the different widths, did i miss it? Are we assuming that this is unchanged from v2? Just asking.. :)

Reply:
yes and I also referred to the text-field component which is already validated for EU.